go-controller/pkg/ovn/secondary_layer3_network_controller_test.go

```diff
@@ -34,12 +35,12 @@ import (
 )
 
 type secondaryNetInfo struct {
-	netName        string
-	nadName        string
-	clustersubnets string
-	hostsubnets    string // not used in layer2 tests
-	topology       string
-	isPrimary      bool
+	netName       string
+	nadName       string
+	clustersubnet string
+	hostsubnet    string // not used in layer2 tests
```

Comment:
here too maybe its intentional cause you want to tell today tests are just supporting 1 family subnet?